pkg/user/logic.go

```diff
@@ -92,7 +96,9 @@ func (b *BusinessLogic) Provision(pr *osb.ProvisionRequest, w http.ResponseWrite
 	exampleInstance := &exampleInstance{ID: pr.InstanceID, Params: pr.Parameters}
 	b.instances[pr.InstanceID] = exampleInstance
 
-	return &osb.ProvisionResponse{}, nil
+	return &osb.ProvisionResponse{
+		Async: b.async,
```

Comment:
We should only set this in the response if AcceptsIncomplete was in the request, otherwise the platform will consider it an error.

Reply:
Makes sense! Will adjust it, thanks.

Reply:
@pmorie Done. PTALA. Thanks!